composer.json

```diff
@@ -22,15 +22,14 @@
         ]
     },
     "require": {
-        "symfony/framework-bundle": "2.*",
-        "symfony/console": "2.*",
+        "symfony/framework-bundle": "^2.7|^3.0",
```

Comment:
Agree, should be `2.8` because that's the LTS version. Thought it was `2.7` up till now, see: http://symfony.com/download